sources/packages/web/src/components/institutions/request-a-change/CompletedOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications?.count">
```

Comment:
Is null check required here for applications?

Reply:
yes, @dheepak-aot , during searching, the application is set to undefined as part of the search bug fix,  probably when we have a stable version, hope we can remove it

Reply:
Thanks for the explanation.